src/test/java/domain/participant/PlayerTest.java

```diff
@@ -0,0 +1,136 @@
+package domain.participant;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import domain.card.Card;
+import domain.card.Denomination;
+import domain.card.Symbol;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import utils.ExceptionMessages;
+
+class PlayerTest {
+
+    private Player player;
+    private Dealer dealer;
+
+    @BeforeEach
+    void init() {
+        player = new Player("test");
+        dealer = new Dealer();
+    }
+
+    @Test
+    @DisplayName("이름이 공백인 경우 예외를 발생한다.")
+    void playerEmptyNameTest() {
+        String name = "";
+
+        assertThatThrownBy(() -> new Player(name))
+            .isInstanceOf(IllegalArgumentException.class)
+            .hasMessage(ExceptionMessages.EMPTY_NAME_ERROR);
+    }
+
+    @Test
+    @DisplayName("제대로 카드를 뽑는지 확인.")
+    void hitCardTest() {
+        Card card = new Card(Symbol.SPADE, Denomination.ACE);
+        player.hit(card);
+
+        assertThat(player.getCards().get(0)).isEqualTo(card);
+    }
```

Comment:
```suggestion
    @Test
    @DisplayName("제대로 카드를 뽑는지 확인.")
    void hitCardTest() {
        Card card = new Card(Symbol.SPADE, Denomination.ACE);
        
        player.hit(card);

        assertThat(player.getCards()).contains(card);
    }
```
contains를 활용하면 인덱스에 영향받지 않고 테스트 할 수 있어요.